apps/package.json

```diff
@@ -113,7 +113,7 @@
     "firebase": "^2.4.2",
     "firebase-bolt": "^0.8.0",
     "firebase-mock": "^1.0.5",
-    "firmata": "0.16.0",
+    "firmata": "^2.2.0",
```

Comment:
brad made [this comment](https://github.com/code-dot-org/code-dot-org/pull/14641/files#r113306384) on a previous PR where it looks like he tried to upgrade firmata to a newer version, but it created an issue with the sensor feature for circuit playground. i could be misunderstanding what he tried / what happened here, but we should do a pass with a CP to make sure the basic features are still working as expected

Reply:
Thanks Maddie! I think in a later comment, Brad wrote that the sensor issue was resolved by adding firmata as an explicit dependency since it's imported in CircuitPlaygroundBoard.js. 
In addition to the unit tests, I really like Brad's manual testing grid he used in the PR, so I'll go ahead and use that as well. I only have a CP Express, not a Classic. I may ping @epeach to help me test the Classic manually. Thank you for the context!